src/main/java/ch/njol/skript/expressions/ExprExperience.java

```diff
@@ -50,16 +51,16 @@
 		"\tif tool of player = diamond pickaxe:",
 		"\t\tadd 100 to dropped experience"})
 @Since("2.1, 2.5.3 (block break event)")
-@Events({"experience spawn", "break / mine"})
+@Events({"experience spawn", "break / mine", "player experience change"})
 public class ExprExperience extends SimpleExpression<Experience> {
 	static {
 		Skript.registerExpression(ExprExperience.class, Experience.class, ExpressionType.SIMPLE, "[the] (spawned|dropped|) [e]xp[erience] [orb[s]]");
```

Comment:
_Just gonna leave this here but it's not related to this PR._
This can be changed in another PR as it allows `xperience` to be used (breaking change)